src/poker/EquityEvaluator.java

```diff
@@ -238,14 +245,34 @@ public List<Double> finalCalculateEquity(GameResult gameResult) {
         double roundedPlayer2Equity = Math.round(player2Equity * 100.0) / 100.0;
         playersEquity.add(roundedPlayer1Equity);
         playersEquity.add(roundedPlayer2Equity);
+        System.out.println(player1.getNamePlayer() + player1.getHandPlayer());
```

Comment:
tak jak pisalem wyzej nie powinno byc w tej klasie zmiennej globalnej Player pewnie dodales to bo w tym miejscu sie printowalo equity wiec wydalo ci sie ze jest to dobre miejsce. Rozwiązaniem powinno byc calkowite wywalenie printow stąd i jak gdzies bedzie potrzeba printa to bedzie dostep do tych wartosci dzieki "return playersEquity;"